test/functional/assertion-helper.js

```diff
@@ -0,0 +1,8 @@
+var expect = require('chai').expect;
+
+
+exports.errorInEachBrowserContains = function errorInEachBrowserContains (testError, message, errorIndex) {

```

Comment:
No. Bad advice. Always try to export named functions. This way you will have proper stack traces